.readthedocs.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+formats: all
+sphinx:
+  configuration: docs/conf.py
+  fail_on_warning: true
+
+build:
+  os: ubuntu-22.10
+  tools:
+    python: "3.7"  # use min version for async-lru==1.0.3
```

Comment:
Why not use recent Pythion to build docs?

Reply:
Right, I was under the impression we need to ensure Python 3.7 builds works correctly. Updated in dc45a2e.